PROJECT

```diff
@@ -15,7 +16,6 @@ resources:
   version: v1alpha1
 - api:
     crdVersion: v1
-    namespaced: false
```

Comment:
was this unnecessary?

Reply:
the kubebuilder tool did it automatically, so basically the default is false.